akka-typed/src/test/scala/akka/typed/adapter/ActorSystemAdapterSpec2.scala

```diff
@@ -0,0 +1,223 @@
+/**
+ * Copyright (C) 2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package akka.typed.adapter
+
+import scala.util.control.NoStackTrace
+
+import akka.{ actor ⇒ untyped }
+import akka.testkit.AkkaSpec
+import akka.typed.ActorRef
+import akka.typed.Behavior
+import akka.typed.Terminated
+import akka.typed.scaladsl.Actor._
+import akka.typed.scaladsl.AskPattern._
+import akka.typed.testkit.scaladsl.TestKitSettings
+import akka.typed.testkit.scaladsl.TestProbe
+
+object ActorSystemAdapterSpec2 {
```

Comment:
several FIXME/TODO in this test describing what is missing, I will continue working on those